### PR TITLE
SDKS-3571 Align Device Management interface with other platforms

### PR DESCRIPTION
**Pull request by witrisna**
# JIRA Ticket

[SDKS-3571](https://pingidentity.atlassian.net/browse/SDKS-3571)

# Description

Align Device Management interface.

```kotlin
//To Retrieve the devices
deviceClient.oath.get()
deviceClient.push.get()
deviceClient.webAuthn.get()
deviceClient.bound.get()
deviceClient.profile.get()

//To Update the devices
deviceClient.webAuthn.update(WebAuthnDevice)
deviceClient.bound.update(BoundDevice)
deviceClient.profile.update(ProfileDevice)

//To Delete the device
deviceClient.oath.delete(OathDevice)
deviceClient.push.delete(PushDevice)
deviceClient.webAuthn.delete(WebAuthnDevice)
deviceClient.bound.delete(BoundDevice)
deviceClient.profile.delete(ProfileDevice)

```

### CODE REVIEW

File: forgerock-auth/src/main/java/org/forgerock/android/auth/selfservice/Device.kt [hunk 1]
```diff
@@ -0,0 +1,165 @@
+/*
+ * Copyright (c) 2024 ForgeRock. All rights reserved.
```
Comment (spetrov) [inline forgerock-auth/src/main/java/org/forgerock/android/auth/selfservice/Device.kt:2]: Do we still use ForgeRock in the copyright header? (I'm not sure...)

**Review by spetrov — APPROVED** [review]
LGTM! 👍🏻

File: forgerock-auth/src/main/java/org/forgerock/android/auth/selfservice/Device.kt (hunk 1 above)
Comment (rodrigoareis) [reply forgerock-auth/src/main/java/org/forgerock/android/auth/selfservice/Device.kt:2]: I believe we should update to `Ping Identity` for any updated file

**Review by rodrigoareis — DISMISSED** [review]
Changes looks good to me

**Review by spetrov — APPROVED** [review]

**Pull request merged**